src/stores/HelpStore/HelpStore.ts

```diff
@@ -71,17 +70,10 @@ export class HelpStore {
         return HelpStore.staticInstance;
     }
 
-    @observable type: HelpType;
-    @observable helpVisible: boolean = false;
-    @observable position: Position = Position.RIGHT;
+    @observable url: HelpURL;
```

Comment:
Perhaps we can remove the observable `url`.

Reply:
ok fixed.